form/src/main/java/org/dhis2/form/ui/Form.kt

```diff
@@ -195,23 +233,89 @@ private fun FieldProvider(
     needToForceUpdate: Boolean,
     textWatcher: TextWatcher,
     coordinateTextWatcher: LatitudeLongitudeTextWatcher,
+    uiEventHandler: (RecyclerViewUiEvents) -> Unit,
```

Comment:
I think that this property is unused, could we lost this interaction?. How are we going to implement it?

Reply:
It is not lost, Text value type does not uses it. When we implement Date in the design sytem it will.